lib.py

```diff
@@ -1,12 +1,35 @@
 from __future__ import annotations
```

Comment:
It's okay to have a directory named "lib" but a `lib.py` file is equivalent to a `stuff.py` -- it's too ambiguous what should go in there and becomes a 9000LOC trash heap too easily.

* setup.py / pypackage.toml
* `src/`
    * `$package_name` -- TBD? perhaps: getsentry_devenv, or if it's meant to help selfhosted too: sentry_devenv
       * `doctor.py`
       * `fixer_lib` -- if we want polyglot support (imo yes), expose each of these via `devenv fixer-lib xyz` subcommands
           * `brew_packages.py`
           * `gitroot.py`

Reply:
I'll divide it into `lib/*.py`.

As for setup.py and package_name - I made it so that devenv isn't installed via pip, it just gets called as a module directly. See the `devenv` executable.